.github/workflows/create-upmerge-PRs.yml

```diff
@@ -0,0 +1,67 @@
+name: Zephyr upmerge
+
+#on:
+#  schedule:
+#    - cron: "0 0 * * *"
+on: workflow_dispatch
+
+env:
+  GH_TOKEN: ${{ secrets.NCS_GITHUB_UPMERGE_TOKEN }}
+  ZEPHYR_UPSTREAM: https://github.com/zephyrproject-rtos/zephyr
+
+jobs:
+  auto-upmerge-create-PRs:
+    if: github.repository == 'nrfconnect/sdk-nrf'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the code
+        uses: actions/checkout@v3
+        with:
+          path: ncs/nrf
+          fetch-depth: 0
+
+      - name: Install base dependencies
+        working-directory: ncs
+        run: |
+          sudo pip3 install -U setuptools wheel pip
```

Comment:
Surely `sudo` should be removed, no? What is going on here?

Reply:
I'm not very familiar with github runners so I'm not sure (so I didn't test without sudo). Basically this part is copy pasted from here: 
https://github.com/nrfconnect/sdk-nrf/blob/main/.github/workflows/docbuild.yml#L45

Reply:
`pip3 install --user and` would not be sufficient here? sudo is needed because pip3 install, by default, in protected dir (/usr/local or something).